esp-hal-common/src/soc/esp32/mod.rs

```diff
@@ -1,3 +1,10 @@
+//! SOC (System-on-Chip) module(ESP32)
```

Comment:
```suggestion
//! SOC (System-on-Chip) module (ESP32)
```

Reply:
This module isn't public, by the way, so maybe doc comments aren't really necessary here. Child modules are re-exported individually.